content/a/prehistory.md

```diff
@@ -0,0 +1,49 @@
+
+After the war, only two shows in 1919 are known to have been held, one in Wrocław and one in Warsaw, with Karl Saft and Stanisław Zbyszko making their returns, and also probably the first appearance of a wrestler fighting under a clear ring name: Cyklop. This marks the end of the prehistoric era of wrestling on the Polish soil.
+
+Following World War II and the restoration of Polish sovereignty on the arena of international politics, Poland held no wrestling events for decades. It wasn’t until the late ‘80s when Andrzej Supron, a Polish Olympic silver medallist in Greco-Roman wrestling, accidentally found out about pro-wrestling when channel surfing during his stay in the United States. After meeting with Ivan Putski, Supron decided to introduce the modern flavour of pro-wrestling to Poland and after returning home, gathered a group of athletes and sportsmen whom he instructed on his own, despite not being a trained pro-wrestler himself. In the early ‘90s the group – known as SupronStars – went on tour of Poland (including Warsaw and Poznań) and the USSR (Georgia, Kazakstan, Armenia, Azerbaijan, Lithuania, Latvia and Moscow, probably more), with the show in Tbilisi drawing up to 14 thousand people in an arena designed for eight thousand. With the tour over, the group disbanded.
+
+In 1993 an independent American group named World Wrestling Superstars, featuring such names as Bob Orton (Randy Orton’s father), The Iron Sheik or Greg Valentine, was touring Europe. Andrzej Supron learned about this and invited them to do a single show in Warsaw, which took place in the Torwar Hall on 14th December 1993 before a crowd of 600 people. The show was covered by Polish newspapers and television.
+
+At the end of the ‘90s or the beginning of 2000, Andrzej Supron was contacted by Paweł “Boryss” Borkowski, who proposed creating the first fully professional Polish pro-wrestling federation, aptly named “Polish Pro-Wrestling Federation”. Despite gathering a group of people (none of whom had any prior contact with wrestling), holding trainings and even importing a professional ring (albeit very late), PPWF held no shows. The initial attempt failed due to very low ticket sales and the second try, in 2003, ended when the sponsor pulled out mere three weeks before the planned show. PPWF folded, marking the end of the pre-modern era of Polish wrestling.

```

Comment:
Some of that is covered in the article for TBW, consider linking it.

Reply:
Tam jest w sumie jeden akapit, do tego skupiający się bardziej na PPWF, więc linkowanie dla tak małej ilości tekstu raczej nie ma sensu (co innego, gdyby o Supronie było pół szpalty); no i czytelniej będzie, jeżeli tekst będzie spójny.